ppr-ui/src/utils/mhr-api-helper.ts

```diff
@@ -309,7 +309,12 @@ export async function submitMhrRegistration (payloadData, queryParamData) {
     }
   }
 }
-
+/**
+ * Method to return Mhr registrations.  
+ * 
```

Comment:
@ChasBoucher 
You must have some extra spaces here being picked up by the linter on lines 313 and 314.

Looks like an extra space AFTER `Mhr registrations.  `
Same on the following line, an extra space after `*  `

Reply:
Removed